### DISCUSSION TITLE
Pymunk play

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#30] [Show and tell]
![sketch_2021_07_23pymunk_on_py5](https://user-images.githubusercontent.com/3694604/126852072-e3c1de02-dfaf-495a-9128-d508b3a71471.gif)

https://github.com/villares/sketch-a-day/blob/main/2021/sketch_2021_07_23pymunk_on_py5/sketch_2021_07_23pymunk_on_py5.py

**Comment by tabreturn** [comment]
Look, mom! Python mode for Processing using third-party Python libraries with C extensions :partying_face: :wink:

**Reply by hx2A** [reply]
That is really cool! Thanks for sharing your explorations! This is really going to open up a lot of new possibilities.
